app/models/institution.rb

```diff
@@ -236,8 +236,8 @@ def self.autocomplete(search_term, limit = 6)
     clause = ['facility_code = :facility_code']
 
     if fuzzy_search
-      clause << 'institution % :upper_search_term'
-      clause << 'city % :upper_search_term'
+      clause << 'SIMILARITY(institution, :search_term) > :name_threshold'
+      clause << 'SIMILARITY(city, :search_term) > :city_threshold'
```

Comment:
Are the various fuzzy search results/solutions still being tested? I remember seeing a PR last week that removed the city clause

Reply:
Yes, we are adding this back in with a much lower institution threshold so that staging searches work correctly

https://github.com/department-of-veterans-affairs/vets-api/pull/4474 split out the timeouts so there is now a 2 second timeout for searches